### DISCUSSION TITLE
Sort apps separately from folders

### DISCUSSION THREAD

**Discussion by dertuxmalwieder** [mikekazakov/nimble-commander#320] [Ideas]
Maybe I just have not found the particular Nimble Commander option yet, but it *seems* that "sorting folders separately" won't work too well, as Nimble Commander just (mis)takes applications (granted, those *are* folders, but special ones) as regular folders:

<img width="660" alt="Bildschirmfoto 2024-07-09 um 20 03 42" src="https://github.com/mikekazakov/nimble-commander/assets/125629/c5a2f619-7ab1-4739-8195-b250086bf041">

MaxCommander - the only other *commander* I use right now - does it as I would expect it:

<img width="781" alt="Bildschirmfoto 2024-07-09 um 20 04 25" src="https://github.com/mikekazakov/nimble-commander/assets/125629/946cba95-38a5-46f0-8257-38224e6492dd">

My mistake, I guess? If it's not, I'd like to see this behavior in Nimble Commander as well.

**Comment by mikekazakov** [comment]
Nimble Commander does exactly what it advertises - it separates folders from regular files, ordering folders before regular files.
It doesn't distinguish bundles (including bundled applications) from other kinds of folders.
I'm not sure how an additional criteria, like being a bundle, can fit into this scheme.

**Reply by dertuxmalwieder** [reply]
Maybe add another option in the View -> Sorting menu? I mean, I could perfectly understand if you won't, because what Nimble Commander does right now is the best kind of correct (= technically correct).

**Reply by mikekazakov** [reply]
I don't have bandwidth for it, but anybody is free to try implementing such feature themselves.
